book/src/chap20.asciidoc

```diff
@@ -184,36 +183,36 @@ julia> cos.(x) == 0.5*(ℯ.^(im*x)+ℯ.^(-im*x))
 true
 ----
 
-Here, another example of the dot operator is shown. Julia also allows numeric literals to be juxtaposed with identifiers as coefficients as in +2π+.
+Aqui, outro exemplo do uso do operador ponto é mostrado. O Julia também permite que literais numéricos sejam justapostos com identificadores e coeficientes, como em +2π+.
 
 === Strings
 
-In <<chap08>> and <<chap09>>, we did some elementary searches in string objects. Julia can handle however Perl-compatible regular expressions (_regexes_), which eases the task of finding complex patterns in string objets.
-(((regex)))
+Em <<chap08>> e <<chap09>>, nós fizemos algumas buscas elementares com objetos string. O Julia pode no entanto, lidar com expressões regulares compatíveis com o Perl, que facilita a tarefa de achar padrões complexos em objetos string.
+(((expressão regular)))
 
-The +usesonly+ function can be implemented as a regex:
-(((usesonly)))(((Regex)))((("type", "Base", "Regex", see="Regex")))(((occursin)))((("function", "Base", "occursin", see="occursin")))
+A função +usa_somente+ pode ser implementada como uma regex(expressão regular):
+(((usa_somente)))(((Regex)))((("type", "Base", "Regex", see="Regex")))(((occursin)))((("função", "Base", "occursin", see="occursin")))
 
 [source,@julia-setup chap20]
 ----
-function usesonly(word, available)
-  r = Regex("[^$(available)]")
-  !occursin(r, word)
+function usa_somente(palavra, disponível)
+  r = Regex("[^$(disponível)]")
+  !occursin(r, palavra)
 end
 ----
 
-The regex looks for a character that is not in the +available+ string and +occursin+ returns +true+ if the pattern is found in +word+.
+A expressão regular procura um caractere que não está na string +disponível+ e +occursin+ retorna +true+ se o padrão é achado em +word+.
 
 [source,@julia-repl-test chap20]
 ----
-julia> usesonly("banana", "abn")
+julia> usa_somente("banana", "abn")
 true
-julia> usesonly("bananas", "abn")
+julia> usa_somente("bananas", "abn")
 false
 ----
 
-Regexes can also be constructed as non-standard string literals prefixed with +r+:
-(((string)))(((match)))((("function", "Base", "match", see="match")))
+Expressões regulares também podem ser construídas como literais strings não padronizadas prefixadas com +r+:
```

Comment:
Você tem que decidir se vai utilizar _**regex**_ ou _**expressões regulares**_. Se for o caso, alterar na linha 193.